src/main/java/org/opentripplanner/routing/trippattern/TripTimes.java

```diff
@@ -240,6 +264,13 @@ public String getHeadsign(final int stop) {
         }
     }
 
+    /**
+     * @return List of via names for a stop.
+     */
```

Comment:
This need more doc here. Info needed is:
 - The purpose of the `via` and what it is used for.
 - What is this in NeTEx and GTFS
 - Legal return values? : `null`, [`null`]. (This belong in the `@return` part, if you want to)
 
 I am not sure if we should use String[] or List<String> for this type. I also is not sure if we should initialize an empty array containing empty arrays for this. The Norwegian graph contains 250 000 instances of this class. If this is not something you have on every transport in Skaane, then maybe we can use `List<String>[]` as a data type for the field and `List<String>` as a return type for  this method. Then if no vias exist we simply do:
 ```
return via == null ? List.of() : via; 
 ```
 Comments on this appreciated!

Reply:
The main reason for why I've chosen 2D array is to use deduplicator. That way only one instance of the array should be created for each combination. Therefore memory use should not increase that much. I do agree that having an empty array of empty arrays is a little bit unnecessary. I've change the code so that the field is set to null in case there are no via/vias registered for stop/stop sequence. 

I've also added more explicit Java Docs.